recipe/meta.yaml

```
@@ -12,13 +12,10 @@ package:
 
 source:
   url: https://github.com/NVIDIA/cccl/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 141801ddddf5b911f64f97be66a7c75f98374bebca60f275bedec75b8eeab345
-  patches:
-    # Disable conda's CMAKE_FIND_ROOT_PATH, which breaks cudax include path discovery
-    - patches/0001-Disable-conda-CMAKE_FIND_ROOT_PATH.patch
+  sha256: e24b56ee796c5777d679facfade2285cdfb8dc4262da8fc3261348c36e21a86
```

I think I made a typo.
```suggestion
  sha256: e24b56ee796c5777d679facfade2285cdfb8dc4262da8fc3261348c36e21a868
```